Source/WebCore/Modules/reporting/ReportBody.h

```diff
@@ -25,23 +25,40 @@
 
 #pragma once
 
-#include <wtf/Ref.h>
+#include <wtf/IsoMalloc.h>
 #include <wtf/RefCounted.h>
 
 namespace WebCore {
 
-class ReportBody : public RefCounted<ReportBody> {
-    WTF_MAKE_FAST_ALLOCATED;
+enum class ReportBodyType : uint8_t {
+    CSPViolation,
+    // More to come
+};
+
+class WEBCORE_EXPORT ReportBody : public RefCounted<ReportBody> {
```

Comment:
Don't we need encode/decode for ReportBody as well?

Reply:
We don't ever encode a ReportBody (which is kind of an abstract class). Right now, we only encode a CSPViolationReportBody, and in the future we will encode a couple of others.